code/API_definitions/device-status.yaml

```diff
@@ -99,7 +99,7 @@ info:
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
-  version: 0.5.0-rc
+  version: 0.5.0-wip
```

Comment:
wip?

Reply:
I'm not familiar with the process, do we have to go back to wip and then make another PR for a release candidate?

Reply:
Yes, the idea is to prepare RC2 after applying changes based on feedback from the latest RC, so the "wip" distinguishes fluid version with ongoing changes from RC.